instance/migrations/0054_merge.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.5 on 2016-06-05 08:50
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('instance', '0051_auto_20160530_1229'),
+        ('instance', '0053_remove_subdomain_basedomain'),
+    ]
+
+    operations = [
+    ]
```

Comment:
@mtyaka I think it would be worth renaming the migrations to linearize them. That would allow us to get rid of this migration (I think).

```
Applying instance.0051_add_external_domains... OK
Applying instance.0052_add_internal_domain_fiels... OK
Applying instance.0053_remove_subdomain_basedomain... OK
Applying instance.0051_auto_20160530_1229... OK
Applying instance.0054_merge... OK
```

vs.

```
Applying instance.0051_auto_20160530_1229... OK
Applying instance.0052_add_external_domains... OK
Applying instance.0053_add_internal_domain_fiels... OK
Applying instance.0054_remove_subdomain_basedomain... OK
```